docs/source/create/entities/zone-entity.rst

```diff
@@ -0,0 +1,166 @@
++------------------------+---------------------------------------------------------------------------------------------+
+| Filter                 | Filters are specialized JavaScript functions that prevent unwanted modifications to         |
+|                        | entities inside the zone. This property takes the URL to such a script.                     |
+|                        | See `Protect Your Domain’s Content <../../host/configure-settings/entity-filters.html>`_    |
+|                        | for more information.                                                                       |
++------------------------+---------------------------------------------------------------------------------------------+
+| Key Light              | This sets a directional light, e.g. a sun, for the entire zone.                             |
++------------------------+---------------------------------------------------------------------------------------------+
+| Skybox                 | This lets you set a so called "Skybox". This is commonly used to apply a sky texture to     |
+|                        | the zone.                                                                                   |
++------------------------+---------------------------------------------------------------------------------------------+
+| Ambient Light          | This is used to apply light coming from the skybox. This is used if you have light coming   |
+|                        | from more than just a sun. For example if you have multiple suns.                           |
++------------------------+---------------------------------------------------------------------------------------------+
+| Haze                   | Creates fog or mist.                                                                        |
++------------------------+---------------------------------------------------------------------------------------------+
+| Bloom                  | This enables a visual effect known as "bloom". A slight bloom effect could be used to make  |
+|                        | zones appear bright.                                                                        |
++------------------------+---------------------------------------------------------------------------------------------+
+| Avatar Priority        | This property changes the update priority of avatar movements by allocating more bandwidth  |
+|                        | which has been reserved by the avatar mixer. It is commonly used on live events to have     |
+|                        | entertainers' avatar movements still be smooth, even with a huge crowd also being updated.  |
++------------------------+---------------------------------------------------------------------------------------------+
+
+^^^^^^^^^
+Key Light
+^^^^^^^^^
+
+Key lighting is used to illuminate a zone. You can think of the key light as a sun which illuminates a zone from a specific direction.
```

Comment:
This seems out of scope of this.

Reply:
The reason I put that there is because we removed the example reference from below. You have a lot of examples so I figured explaining that there might help summarize what you can do with these values? Dunno.

Reply:
Telling people to think about the key lighting as a sun seems like enough of an example already.

Reply:
I see what you mean. Alright, then I suppose removing the example from the "Key Light Color" and leaving this as is might be most appropriate?

Reply:
I wouldn't want to remove the Key Light Color example. The point of that is to give people an idea what they can actually do with that property. Personally, when I first looked at at the Key Light Color property I thought to myself "Why would I ever want to change the color."